cg/store/crud/create.py

```diff
@@ -260,25 +259,6 @@ def relate_sample(
         new_record.father = father
         return new_record
 
-    def add_flow_cell(
```

Comment:
Isn't removing this breaking a lot of tests? maybe we could leave it for another PR